recipes/graphdatascience/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "graphdatascience" %}
+{% set version = "1.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/graphdatascience-{{ version }}.tar.gz
+  sha256: 7699b15417a923c89c5b7f07ffb4e073141f6cf711b0082cb82341fcee968101
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=42
+    - wheel
+    - pip
+  run:
+    - python >=3.8
+    - multimethod >=1.0,<2.0
+    - neo4j-python-driver >=4.4.2,<6.0
+    - pandas >=1.0,<3.0
+    - pyarrow >=11.0,<16.0
+    - textdistance >=4.0,<5.0
+    - tqdm >=4.0,<5.0
+    - typing-extensions >=4.0,<5.0
+    - requests
+
+test:
+  imports:
+    - graphdatascience
+  commands:
+    - pip check
+  requires:
+    - pip
+
+outputs:
+  - name: graphdatascience
+
+  - name: graphdatascience.ogb
```

Comment:
Please add import tests to the following two outputs.

Reply:
I've added the import tests, my apologies for using the wrong @ regarding the PRs